conf/protein_decoy.yaml

```diff
@@ -9,7 +9,6 @@ proteindb_decoy:
   keep_target_hits: true
```

Comment:
If those params were part of the pipeline you could validate and display them in a UI. But could be done in a future version.